tensorboard/tools/build_lint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+ret_val=0
+
+for build_path in "$@"
+do
+  # We do not care about few lint warns just yet.
+  buildifier --mode=check --lint warn "$build_path" 2>&1 | grep -v -E "(load-on-top:|native-py:|native-java:|constant-glob:)" 1>&2
+  if [ "$?" -eq 0 ]; then
+    ret_val=$((1 + $ret_val))
+  fi
+done
+
+exit $ret_val
```

Comment:
Having a “count of failures” exit code is fairly common and certainly
reasonable, but when you do this you need to cap it at some value
because return codes wrap around after 256, so having exactly 256 errors
would be misclassified as passing. (Often I see this capped at 100 or
99; it’s good form to not return higher than 127, as those error codes
are also used by the shell to indicate signals.)

Reply:
Yeah, I did not think about that! Thanks for catching it. Unfortunately, it is moot now :)